modules/ROOT/pages/tokenization-logs.adoc

```diff
@@ -43,6 +43,30 @@ Example tokenization summary message:
  }
 }
 
+== Tokenization Audit Logging
+
+Audit logs are generated when a tokenization or detokenization request is initiated. To enable audit logs, select the *Enable Audit Logs* setting.
```

Comment:
But...you have to enable these before they are generated, so why are these sentences in this order?
Also, note that the second sentence reads like a step in a task, not text for a concept topic.

Reply:
I wonder if, instead of making these subtopics, the content here belongs in an already existing topic or topics in this set? Hard to tell without context. I'm sure you know best.

Reply:
Only audit logging has to be enabled this way (through the UI)--it's separate from the preceding section.